lib/index.js

```diff
@@ -87,6 +87,11 @@ function _createUrl(config) {
   return url;
 }
 
+function _sanitizeUrl(url) {
```

Comment:
Please move helper to bottom of the file along with any other helpers (if any).  Private functions like this are hoisted and available before they are defined.

Reply:
Fixed - df652a8